CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 - PR #857: Small modifications to comms for utilizing IB w/ Dask
 - PR #851: Random forest Stateless API wrappers
 - PR #895: Pretty prints arguments!
+- PR #911: Thanks to CannyLab's excellent BH TSNE implementation!
```

Comment:
I would reprhase this to "Update headers to credit CannyLabs BH TSNE implementation". I tend to be boring in my changelogs ;)